cmp/tab/TabContainerModel.js

```diff
@@ -86,27 +88,18 @@ export class TabContainerModel {
                 return;
             }
 
+            if (persistWith) {
+                console.warn('Active tab persistance is ignored when a tab container uses routing.');
```

Comment:
makes sense -- perhaps mention in the docstring, for persistWith.  Also spelling of `persistence`

Reply:
Changed to throwIf up top as suggested.  Spelling no longer an issue.